### DISCUSSION TITLE
Hello author, may I ask when this article will be officially published?

### DISCUSSION THREAD

**Discussion by laohutie** [jamesjiadazhan/dietaryindex#221] [Q&A] [answered]
Hello author, may I ask when this article will be officially published?

**Comment by jamesjiadazhan** [comment, marked as answer]
We are revising the manuscript submitted to the American Journal of Clinical Nutrition. Major revision was just completed and submitted for additional feedback.

**Comment by laohutie** [comment]
Congratulations to the teacher, thank you for your reply

**Comment by jamesjiadazhan** [comment]
You can quote the preprint too. Thanks!

Zhan JJ, Hodge RA, Dunlop AL, et al. Dietaryindex: A User-Friendly and Versatile R Package for Standardizing Dietary Pattern Analysis in Epidemiological and Clinical Studies. Preprint. bioRxiv. 2023;2023.08.07.548466. Published 2023 Aug 9. doi:10.1101/2023.08.07.548466

https://pubmed.ncbi.nlm.nih.gov/37609152/

**Comment by jamesjiadazhan** [comment]
FYI, dietaryindex is published on the American Journal of Clinical Nutrition (online only currently)!
- https://doi.org/10.1016/j.ajcnut.2024.08.021

Please cite the dietaryindex using the official reference on AJCN.
